addons/mail/static/tests/message/link_preview_test.js

```diff
@@ -261,6 +261,13 @@ QUnit.test("No crash on receiving link preview of non-known message", async (ass
     env.services.rpc("/mail/link_preview", { message_id: messageId });
     await nextAnimationFrame();
     assert.ok(true);
+    // simulate receiving deletion of link preview
+    pyEnv["bus.bus"]._sendone(pyEnv.currentPartnerId, "mail.link.preview/delete", {
```

Comment:
Why not actually call the remove link preview route?

I think the explained scenario is not correct, it's probably not after a deleted message, but just receiving the notification on a tab that didn't load the message in the first place.

I think it should be relatively easy to make a new test without simulating the bus.

Reply:
Done (calling RPC to delete link preview, and updated commit message accordingly)